### PR TITLE
Fixed viewAnimationDuration is not updated if changed in InterfaceBuilder

### PR DESCRIPTION
**Pull request by JamesJSY**

### CODE REVIEW

File: Pod/Classes/UI/CardTextField.swift [hunk 1]
```diff
@@ -217,9 +217,9 @@ public class CardTextField: UITextField, NumberInputTextFieldDelegate {
         monthTextField?.deleteBackwardCallback = {_ -> Void in self.numberInputTextField?.becomeFirstResponder()}
         yearTextField?.deleteBackwardCallback = {_ -> Void in self.monthTextField?.becomeFirstResponder()}
         
-        let textFields: [UITextField?] = [numberInputTextField,cvcTextField,monthTextField,yearTextField]
+        let textFields: [UITextField?] = [numberInputTextField, cvcTextField, monthTextField, yearTextField]
         textFields.forEach({
-            $0?.keyboardType = .NumberPad
+            $0?.keyboardType = keyboardType
```
Comment (DannyVancura) [inline Pod/Classes/UI/CardTextField.swift]: I'd probably not make this customizable - developers will have to select NumberPad in interface builder whenever they integrate one and I don't see a chance that anything but a NumberPad keyboard will be required here.

**Comment by Haud** [comment]
Any update on this from @DannyVancura 's comments?

**Comment by JamesJSY** [comment]
We are waiting for more thoughts on this from people. If the design is to offer user ability to customize the keyboard, this pr should be merged. But on other thoughts, it doesn't make sense to provide keyboards of other kind, other than NumberPad, when only numbers are needed in the textfield. So it might be better to just disable keyboard customizability. I prefer the latter one.

**Comment by Haud** [comment]
I agree; unless we find a use case where a user can enter letters into any of the fields, this feels like a superflous change.

**Comment by JamesJSY** [comment]
Removed the ability to customize keyboard type. This branch only solves the viewAnimationDuration issue now. @Haud

**Comment by DannyVancura** [comment]
I added a fix to let this property properly show up in Interface Builder.

👍

**Pull request merged**